cypress/e2e/settings/circulation/general/edit-circulation-rules.cy.js

```diff
@@ -70,9 +67,50 @@ describe('ui-circulation-settings: Edit circulation rules', () => {
     Users.deleteViaApi(newUserId);
   });
 
+  it(
+    'C650: Test adding fallback policies (loan, request, notice, overdue, lost item) (vega) (TaaS)',
+    { tags: [TestTypes.criticalPath, DevTeams.vega, Parallelization.nonParallel] },
+    () => {
+      // Delete Circulation Rules
+      CirculationRules.clearCirculationRules();
+      // Fill in priority
+      CirculationRules.fillInPriority();
+
+      // Enter fallback-policy:
+      CirculationRules.fillInCirculationRules('fallback-policy: ');
+      CirculationRules.moveCursorFocusToTheEnd();
+
+      // A dropdown list of policy types is displayed
+      // Choose Lost item fee policies and choose a value for that policy
+      CirculationRules.clickCirculationRulesHintItemForPolicyType('Lost item fee policies');
+      CirculationRules.clickCirculationRulesHintItem(
+        LOST_ITEM_FEES_POLICY_NAMES.LOST_ITEM_FEES_POLICY,
+      );
+
+      // Choose Loan policies and choose a value for that policy
+      CirculationRules.clickCirculationRulesHintItemForPolicyType('Loan policies');
+      CirculationRules.clickCirculationRulesHintItem(LOAN_POLICY_NAMES.EXAMPLE_LOAN);
+
+      // Choose Overdue fine policies and choose a value for that policy
+      CirculationRules.clickCirculationRulesHintItemForPolicyType('Overdue fine policies');
+      CirculationRules.clickCirculationRulesHintItem(OVERDUE_FINE_POLICY_NAMES.OVERDUE_FINE_POLICY);
+
+      // Choose Request policies and choose a value for that policy
+      CirculationRules.clickCirculationRulesHintItemForPolicyType('Request policies');
+      CirculationRules.clickCirculationRulesHintItem(REQUEST_POLICY_NAMES.ALLOW_ALL);
+
+      // Choose Patron notice policies and choose a value for that policy
+      CirculationRules.clickCirculationRulesHintItemForPolicyType('Patron notice policies');
+      CirculationRules.clickCirculationRulesHintItem(NOTICE_POLICY_NAMES.SEND_NO_NOTICES);
+
```

Comment:
I think here need expected result

Reply:
verification is inside `clickCirculationRulesHintItem` and `clickCirculationRulesHintItemForPolicyType` methods